src/flags.ts

```diff
@@ -0,0 +1,34 @@
+/*********************************************************************
```

Comment:
Maybe this file should be better named: `common-flags.ts`?

Reply:
Fixed.